.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+- "2.7"
+- "3.4"
+- "3.5"
+install:
+- pip install -r requirements-test.txt
+script:
+- python manage.py collectstatic --no-input
+- python manage.py migrate
+- python manage.py test -v 2
```

Comment:
We are still fixing the funcional tests, so for now, let's just run the unit tests (`python manage.py test lists -v 2`)